### ISSUE TITLE
Internal Server Error or failed login when logging in to /reports

### ISSUE THREAD

**Issue by seanh** [hypothesis/lms#68]
Go to <https://qa-lms.hypothes.is/reports> and login with the correct username and password, and you'll get a 500 Internal Server Error.

https://sentry.io/hypothesis/lms-backend/issues/419033074/

```
HTTPNotFound: The resource could not be found.
  File "pyramid/tweens.py", line 12, in _error_handler
    response = request.invoke_exception_view(exc_info)
  File "pyramid/view.py", line 756, in invoke_exception_view
    raise HTTPNotFound

TypeError: can't concat bytes to NoneType
(24 additional frame(s) were not displayed)
...
  File "pyramid/authentication.py", line 621, in remember
    return self.cookie.remember(request, userid, **kw)
  File "pyramid/authentication.py", line 992, in remember
    cookie_value = ticket.cookie_value()
  File "pyramid/authentication.py", line 676, in cookie_value
    v = '%s%08x%s!' % (self.digest(), int(self.time),
  File "pyramid/authentication.py", line 673, in digest
    self.user_data, self.hashalg)
  File "pyramid/authentication.py", line 752, in calculate_digest
    tokens + b'\0' + user_data)

TypeError: can't concat bytes to NoneType
```

If I try to login to <https://lms.hypothes.is/reports> (production rather than qa) then an attempt to login with the correct username and password just reloads the login page with a failed login message.

**Comment by segdeha** [comment]
@nick-benoit14 Can you comment as to why this is now closed? Maybe link to whatever PR fixed the issue?

**Comment by nick-benoit14** [comment]
@segdeha Sure thing. 
Fixed by #71
